### PR TITLE
[Mailer][MailJet] Fix parameters for TrackClicks and TrackOpens

### PR DESCRIPTION
**Pull request by torohill**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Issues        | N/A
| License       | MIT

The current parameters to track clicks and opens don't do anything because they are named incorrectly. Update so that they actually work.

See API reference: https://dev.mailjet.com/email/reference/send-emails/

### CODE REVIEW

File: src/Symfony/Component/Mailer/Bridge/Mailjet/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add `RemoteEvent` and `Webhook` support
+ * Fix `TrackClicks` and `TrackOpens` parameters
```
Comment (xabbuh) [inline src/Symfony/Component/Mailer/Bridge/Mailjet/CHANGELOG.md]: This line must be reverted, changelogs for patch releases are drafted automatically. The components' changelog files do only contain features/deprecations added in minor releases.

Comment (torohill) [reply src/Symfony/Component/Mailer/Bridge/Mailjet/CHANGELOG.md]: I've reverted the line in the changelog.

**Review by xabbuh — APPROVED** [review]

**Comment by OskarStark** [comment]
But the header name itself isn't changed and the names are just renamed, is there any logic I don't see?

Tests pass before, tests pass now 🤔

**Comment by torohill** [comment]
@OskarStark The previous `TrackClick` and `TrackOpen` parameters are not actually used by Mailjet, so they weren't doing anything. They need to be passed as `TrackClicks` and `TrackOpens` to Mailjet to configure click and open tracking.. See the API docs: https://dev.mailjet.com/email/reference/send-emails/ 

I guess they were never tested against the live Mailjet API to check they were working as intended. I have tested sending of real emails and with the change these parameters are now working as expected.

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @torohill.

**Pull request merged**